packages/ui/src/components/organisms/DynamicForm/components/RSJVInputAdaters/PhoneInputAdapter/PhoneInputAdapter.tsx

```
@@ -7,17 +7,20 @@ export const PhoneInputAdapter: RJSFInputAdapter = ({
   formData,
   disabled,
   testId,
+  uiSchema,
   onChange,
   onBlur,
 }) => {
+  const { defaultCountry = 'us' } = uiSchema || {};
+
   const handleBlur = useCallback(() => {
     // @ts-ignore
     onBlur && onBlur(id, formData);
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)